test/test_data/blocks_configs.py

```diff
@@ -1,149 +1,220 @@
-        {
-            "id": 13,
-            "type": "image",
-            "props": {
-                "image_mode": "RGB",
-                "shape": None,
-                "source": "upload",
-                "tool": "editor",
-                "default_value": None,
-                "name": "image",
-                "label": None,
-                "css": {},
-            },
-        },
-        {
-            "id": 14,
-            "type": "json",
-            "props": {"default_value": '""', "name": "json", "label": None, "css": {}},
-        },
-        {
-            "id": 15,
-            "type": "button",
-            "props": {
-                "default_value": "Run",
-                "name": "button",
-                "label": None,
-                "css": {},
-            },
-        },
-        {
-            "id": 16,
-            "type": "textbox",
-            "props": {
-                "lines": 1,
-                "placeholder": None,
-                "default_value": "",
-                "name": "textbox",
-                "label": None,
-                "css": {},
-            },
-        },
-    ],
-    "theme": "default",
-    "layout": {
-        "id": 0,
-        "children": [
-            {"id": 1},
-            {"id": 2},
-            {
-                "id": 3,
-                "children": [
-                    {
-                        "id": 5,
-                        "children": [
-                            {"id": 6, "children": [{"id": 7}, {"id": 8}]},
-                            {"id": 9},
-                        ],
-                    },
-                    {
-                        "id": 5,
-                        "children": [
-                            {"id": 6, "children": [{"id": 7}, {"id": 8}]},
-                            {"id": 9},
-                        ],
-                    },
-                    {
-                        "id": 11,
-                        "children": [
-                            {"id": 12, "children": [{"id": 13}, {"id": 14}]},
-                            {"id": 15},
-                        ],
-                    },
-                    {
-                        "id": 11,
-                        "children": [
-                            {"id": 12, "children": [{"id": 13}, {"id": 14}]},
-                            {"id": 15},
-                        ],
-                    },
-                ],
-            },
-            {"id": 16},
-        ],
-    },
-    "dependencies": [
-        {"targets": [9], "trigger": "click", "inputs": [2, 7], "outputs": [8]},
-        {"targets": [15], "trigger": "click", "inputs": [2, 13], "outputs": [14]},
-    ],
-}
+	'mode': 'blocks',
+	'components': [{
+		'id': 1,
+		'type': 'markdown',
+		'props': {
+			'default_value': '<pre><code>        # Detect Disease From Scan\n        With this model you can lorem ipsum\n        - ipsum 1\n        - ipsum 2\n</code></pre>\n',
```

Comment:
I got the config from from terminal with `print(xray_blocks.get_config_file())`, not sure why it does not comply with the test.
Thanks for the fix!

Reply:
Same lol